src/ft2font_wrapper.cpp

```diff
@@ -553,10 +553,11 @@ static int PyFT2Font_init(PyFT2Font *self, PyObject *args, PyObject *kwds)
     PyObject *fname;
     FT_Open_Args open_args;
     long hinting_factor = 8;
-    const char *names[] = { "filename", "hinting_factor", NULL };
+    int kerning_factor = 0;
+    const char *names[] = { "filename", "hinting_factor", "_kerning_factor", NULL };
 
     if (!PyArg_ParseTupleAndKeywords(
-             args, kwds, "O|l:FT2Font", (char **)names, &fname, &hinting_factor)) {
+             args, kwds, "O|li:FT2Font", (char **)names, &fname, &hinting_factor, &kerning_factor)) {
```

Comment:
do you want to make _kerning_factor keyword-only? (you can use the "$" specifier for that) otherwise it effectively becomes public API as you can pass it positionally...

Reply:
Sure, let's do that.